Source/town.h

```diff
@@ -3,15 +3,28 @@
  *
  * Interface of functionality for rendering the town, towners and calling other render routines.
  */
+

```

Comment:
There should not be new changes in lines you did not intend to change:
```suggestion
```

Reply:
Ok.